src/main/java/eu/dissco/core/digitalspecimenprocessor/service/HandleService.java

```diff
@@ -151,20 +376,11 @@ private int toDigit(char hexChar) {
   }
 
   private void updateHandle(String id, DigitalSpecimen digitalSpecimen) {
-    var handleAttributes = updatedHandles(digitalSpecimen);
+    var handleAttributes = fillFdoRecordSpecimenAttributes(digitalSpecimen);
```

Comment:
This might impact a check in the processing service as well where we check if the handle needs to be updated. Did you check it, no changes needed there?

Reply:
The issue is that the updateHandle method is now only called when the type has been updated. However, as we changed the FDO Profile with more fields the method in the Handle Service needs to check for more fields as well.